modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -18,28 +18,28 @@ You can add an object storage bucket as a replication repository to the CAM web
 . In the *Replication repositories* section, click *Add repository*.
 . Select a *Storage provider type* and fill in the following fields:
 
-* *AWS* for AWS S3 and generic S3 providers:
+* *AWS* for AWS S3, MCG, and generic S3 providers:
 
-** *Repository name*: Specify the repository name in the CAM web console.
-** *S3 bucket name*: Specify the name of the S3 bucket.
-** *S3 bucket region*: Specify the S3 bucket region. *Required* for AWS S3. *Optional* for a generic S3 provider.
-** *S3 endpoint*:  Specify the URL of the S3 service, not the bucket, for example, `\https://<s3-storage.apps.cluster.com>`. *Required* for a generic S3 provider.
+** *Replication repository name*: Specify the replication repository name in the CAM web console.
+** *S3 bucket name*: Specify the name of the S3 bucket you created.
+** *S3 bucket region*: Specify the S3 bucket region. *Required* for AWS S3. *Optional* for other S3 providers.
+** *S3 endpoint*: Specify the URL of the S3 service, not the bucket, for example, `\https://<s3-storage.apps.cluster.com>`. *Required* for a generic S3 provider. You must use the `https://` prefix.
 
-** *S3 provider access key*: Specify the `<AWS_SECRET_ACCESS_KEY>`.
-** *S3 provider secret access key*: Specify the `<AWS_ACCESS_KEY_ID>`.
+** *S3 provider access key*: Specify the `<AWS_SECRET_ACCESS_KEY>` for AWS or the S3 provider access key for MCG.
+** *S3 provider secret access key*: Specify the `<AWS_ACCESS_KEY_ID>` for AWS or the S3 provider secret access key for MCG.
 ** *Require SSL verification*: Clear this check box if you are using a generic S3 provider.
 
 * *GCP*:
 
-** *Repository name*: Specify the repository name in the CAM web console.
+** *Replication repository name*: Specify the repository name in the CAM web console.
 ** *GCP bucket name*: Specify the name of the GCP bucket.
 ** *GCP credential JSON blob*: Specify the string in the `credentials-velero` file.
 
 * *Azure*:
 
-** *Repository name*: Specify the repository name in the CAM web console.
+** *Replication repository name*: Specify the repository name in the CAM web console.
```

Comment:
The change in line 23 of migration-adding-replication-repository-to-cam.adoc is clearer.  It is better to change 34 as the same way.
```
 *Replication repository name*: Specify the replication repository name in the CAM web console
```